pkg/config/default.go

```diff
@@ -0,0 +1,17 @@
+package config
+
+var DefaultNetworks = Networks{
+	Default: "stub",
+	All: map[string]*Network{
```

Comment:
Maybe we could enforce type safety of the key here by declaring `type NetworkName string`? Not sure if we need it though.